php/tower/.env

```diff
@@ -0,0 +1,33 @@
+APP_NAME=Tower
```

Comment:
is this configuration file useful in such case ? at least could it be set to bare minimum ?